docs/competition/livestream/matches.md

```diff
@@ -0,0 +1,30 @@
+# Matches
+
+One of the most common things broadcast during livestreams is matches. These may come either from some live camera feeds of the arena(s), or pre-recorded video from the simulator.
+
+## Overlay
+
+Whilst matches are playing, we show an [overlay](https://github.com/srobo/livestream-overlay) on top of the videos to present some additional information:
+
+- The match number of the current match
+- The teams competing in the current match (TLA and team name)
+
+And outside of matches, it also shows:
+
+- Current league schedule
+- Times until the next match
+- Upcoming match information
+
+The overlay live updates with changes as they happen. It may occasionally get out of sync, in which case just needs a simple refresh.
+
+The overlay is brought into OBS as a webpage. To ensure content behind it can be shown a [chroma key filter](https://obsproject.com/wiki/Filters-Guide#color-key-and-chroma-key), and the appropriate similarity and smoothness values found until the background is transparent, but the rest of the colours are still showing correctly.
+
+The overlay should be present on all scenes which display matches, and perhaps a few others depending on context.
+
+## Video Scheduler
+
+In a simulator-based competition, matches are pre-recorded. To assist with the automatic queueing and playback of matches in line with the competition schedule, we have a [video scheduler](https://github.com/RealOrangeOne/livestream-video-scheduler).
+
+The video scheduler is currently a fork of the [overlay](#overlay), with much of the functionality removed. Therefore, it's brought into OBS in the same way, just without the chroma key.
+
+Due to issues with the OBS browser, some videos will not play directly through it. In that case the scheduler will either need to be opened in an external browser and screen captured, or the videos transcoded into an alternative format. This is an issue actively being looked into.
```

Comment:
Can we record which types of video, how to detect this and what alternative formats to try?

Reply:
If we're getting into specifics, it's worth mentioning which operating systems are/aren't affected.

Reply:
https://github.com/srobo/runbook/pull/95/commits/510123f071bbf9731a9a7471dabfff1357c5b7e6